src/configuration/configuration.ts

```diff
@@ -4,6 +4,7 @@ import pm2 from "@pm2/io";
 import type Counter from "@pm2/io/build/main/utils/metrics/counter";
 import type Gauge from "@pm2/io/build/main/utils/metrics/gauge";
 import { Scraper } from "@the-convocation/twitter-scraper";
+import * as fs from "fs/promises";
```

Comment:
Might be wise only importing the required methods instead of relying or wildcard

Reply:
A habit of always exposing `fs` (and `path`) in full, but I'll just import `readFile`